README.md

```diff
@@ -3,7 +3,9 @@ Spotify for Nuvola Player 3
 
 Integration of Spotify into your linux desktop via
 [Nuvola Player](https://github.com/tiliado/nuvolaplayer).
- 
+
+Based on the Nuvola Player 2 Spotify plugin by Chase Colman (<chase@colman.io>).
```

Comment:
Good point. I only used his script as a reference to see how to get information from Spotify's API. I'm not familiar with licensing stuff at all, should the original copyright be retained?